app/models/post.rb

```diff
@@ -3,6 +3,23 @@ class Post < ApplicationRecord
   has_many :posts
   has_many :comments
 
-  scope :update_post_counter, -> { User.find(2).update(posts_counter: User.find(2).posts.count) }
-  scope :post_recent_comments, -> { Comment.where(post_id: 2).order(created_at: :desc).first(5) }
+  scope :update_posts_counter, -> { User.find(2).update(postsCounter: User.find(2).posts.count) }
+  scope :post_recent_comments, -> { Comment.where(post_id: 1).order(created_at: :desc).limit(5) }
```

Comment:
- [ ] Let's remove these scopes. We already have `recent_comments` and `update_post_counter` methods that do what we intended for these queries.